lens-cube/src/main/java/org/apache/lens/cube/parse/UnionCandidate.java

```diff
@@ -141,8 +149,19 @@ private String getToString() {
       builder.append(candidate.toString());
       builder.append(", ");
     }
-    builder.delete(builder.length()-2, builder.length());
+    builder.delete(builder.length() - 2, builder.length());
     builder.append("]");
     return builder.toString();
   }
+
+  private List<TimeRange> checkAndGetRangeForChildren(TimeRange timeRange) {
```

Comment:
Is this method Checking anything ? 
Should we call it splitTimeRangeForChildren()